### PR TITLE
Improve azure help.

### PR DESCRIPTION
**Pull request by frankban**
We should switch to using the interactive system for adding credentials, but for the time being at least point the user to a page with useful instructions.

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6606/

File: jujugui/static/gui/src/app/views/utils.js [hunk 1]
```diff
@@ -1659,13 +1659,14 @@ YUI.add('juju-view-utils', function(Y) {
         message: (
           <p>
             The following fields require your Windows Azure management
-            information. For more information please see:&nbsp;
+            information. For instructions on retrieving the required
+            information please see:&nbsp;
             <a className="deployment-panel__link"
-              href="https://msdn.microsoft.com/en-us/library/windowsazure"
+              href="https://jujucharms.com/docs/2.0/help-azure#manually-adding-credentials"
```
Comment (hatched) [inline jujugui/static/gui/src/app/views/utils.js]: I think we should just link them to the top of this page - It doesn't require them to install a bunch of additional dependencies and just use the Juju CLI which is a much better experience than the 'manual' version.

File: jujugui/static/gui/src/app/views/utils.js [hunk 2]
```diff
@@ -1659,13 +1659,14 @@ YUI.add('juju-view-utils', function(Y) {
...
               target="_blank">
-              https://msdn.microsoft.com/en-us/library/windowsazure
+              the Azure Juju documentation
             </a>
-            &nbsp;for details.
+            .
```
Comment (hatched) [inline jujugui/static/gui/src/app/views/utils.js]: "After following the interactive setup copy the values from the `.local/share/juju/credentials.yaml` into this form."

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6607/

File: jujugui/static/gui/src/app/views/utils.js (hunk 1 above)
Comment (frankban) [inline jujugui/static/gui/src/app/views/utils.js]: Included both interactive and manual instructions.

File: jujugui/static/gui/src/app/views/utils.js (hunk 2 above)
Comment (frankban) [inline jujugui/static/gui/src/app/views/utils.js]: Found a better way to do that.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6608/

**Comment by frankban** [comment]
Thanks for the reviews!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**